@xen-orchestra/fs/src/abstract.js

```diff
@@ -117,10 +117,14 @@ export default class RemoteHandlerAbstract {
     return prefix === '/' ? this : new PrefixWrapper(this, prefix)
   }
 
-  async closeFile(fd: FileDescriptor): Promise<void> {
+  async __closeFile(fd: FileDescriptor): Promise<void> {
```

Comment:
As you can see, this class is split in sections, each sections containing methods intended to be use in a specific way, I think you should do this for these methods too.